ui/components/institutional/transaction-failed-modal/transaction-failed-modal.js

```diff
@@ -17,13 +17,13 @@ import {
   Text,
   Box,
   Modal,
-  ModalContent,
-  ModalHeader,
   ModalOverlay,
   Button,
   BUTTON_VARIANT,
   BUTTON_SIZES,
 } from '../../component-library';
+import { ModalContent } from '../../component-library/modal-content/deprecated';
+import { ModalHeader } from '../../component-library/modal-header/deprecated';
 
 const TransactionFailedModal = ({
   hideModal,
```

Comment:
Screenshots showing no visual changes to UI

### Before
<img width="1512" alt="Screenshot 2023-12-10 at 3 24 15 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/d0ebb967-fbf5-48f7-ac0a-edfee4b6f454">


### After
<img width="1512" alt="Screenshot 2023-12-10 at 3 24 27 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/7c4e4838-a46a-43b1-99fa-b830b2e5f94e">